test/EndToEnd/tests/BuildIntegratedTest.ps1

```diff
@@ -216,7 +216,7 @@ function Test-BuildIntegratedProjectClosureWithLegacyProjects {
 function Test-BuildIntegratedMixedLegacyProjects {
     # Arrange
     $project1 = New-ClassLibrary
-    $project1 | Install-Package Newtonsoft.Json -Version 5.0.6
+    $project1 | Install-Package Newtonsoft.Json -Version 13.0.1
```

Comment:
Is it easy to declare a version variable so this can be updated in one place next time if needed?